### PR TITLE
Prepare release r1.1

### PR DESCRIPTION
**Pull request by akoshunyadi**
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:

Prepare the release 1.1

### CODE REVIEW

**Comment by hdamker** [comment]
@akoshunyadi Could you please change the release number to r1.1? See "API release numbers start at r1.1" within https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API_Release_Guidelines.md 

I know that the sub project have done a release r0.6 previously, but now it is time to start into the numbering of the Fall24 release cycle ;-) r1.2 could also be an option, to show that it is already the second release within the current cycle.

File: README.md
Comment (hdamker) [inline README.md:34]: There is no device-status.yaml anymore ... how about the following (lot of source code, but looks quite nice when rendered):

```suggestion
The release r1.1 contains the following API definitions (with inline documentation):
- device-roaming-status [[YAML]](https://github.com/camaraproject/DeviceStatus/blob/r1.1/code/API_definitions/device-roaming-status.yaml)
[[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceStatus/r1.1/code/API_definitions/device-roaming-status.yaml&nocors)
[[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceStatus/r1.1/code/API_definitions/device-roaming-status.yaml)
- device-roaming-status-subscriptions [[YAML]](https://github.com/camaraproject/DeviceStatus/blob/r1.1/code/API_definitions/device-roaming-status-subscriptions.yaml)
[[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceStatus/r1.1/code/API_definitions/device-roaming-status-subscriptions.yaml&nocors)
[[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceStatus/r1.1/code/API_definitions/device-roaming-status-subscriptions.yaml)
- device-reachability-status [[YAML]](https://github.com/camaraproject/DeviceStatus/blob/r1.1/code/API_definitions/device-reachability-status.yaml)
[[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceStatus/r1.1/code/API_definitions/device-reachability-status.yaml&nocors)
[[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceStatus/r1.1/code/API_definitions/device-reachability-status.yaml)
- device-reachability-status-subscriptions [[YAML]](https://github.com/camaraproject/DeviceStatus/blob/r1.1/code/API_definitions/device-reachability-status-subscriptions.yaml)
[[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceStatus/r1.1/code/API_definitions/device-reachability-status-subscriptions.yaml&nocors)
[[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceStatus/r1.1/code/API_definitions/device-reachability-status-subscriptions.yaml)
```

BTW a hint: if you create temporarily a branch `r1.1` you can test the links ... you just need to delete it before you can create the tag ;-)

File: CHANGELOG.md
Comment (hdamker) [inline CHANGELOG.md:3]:
```suggestion
- [r1.1](#r11)
```

**Review by bigludo7 — DISMISSED** [review]
LGTM

**Comment by hdamker** [comment]
@akoshunyadi consider to add @camaraproject/release-management_maintainers as reviewers if the PR is there.

**Comment by akoshunyadi** [comment]
> @akoshunyadi consider to add @camaraproject/release-management_maintainers as reviewers if the PR is there.

yes, I would wait until all PRs are merged, and the change-log is final

**Review by bigludo7 — APPROVED** [review]
LGTM

**Review by hdamker — CHANGES_REQUESTED** [review]
@akoshunyadi Done the review on behalf of Release Management in context of https://github.com/camaraproject/ReleaseManagement/issues/57

Still found some points ;-) 

Please update also the release tracker pages at https://wiki.camaraproject.org/display/CAM/DeviceStatus+API+Release+Tracking

File: CHANGELOG.md
Comment (akoshunyadi) [reply CHANGELOG.md:15]: I understood this like the alpha release was the first, which had all changes and this is the 2nd release of the v0.6....
But I can add the issues here too.

File: code/API_definitions/device-reachability-status-subscriptions.yaml
Comment (akoshunyadi) [reply code/API_definitions/device-reachability-status-subscriptions.yaml:71]: sure, I have never heard about it so far :-)

**Review by hdamker — APPROVED** [review]
LGTM from ReleaseManagement view - suppose we are done here.

@camaraproject/device-status_codeowners  the PR needs a new codeowner review before it can be merged

**Review by maxl2287 — APPROVED** [review]
LGTM

File: CHANGELOG.md
Comment (eric-murray) [inline CHANGELOG.md:32]:
```suggestion
Note: the previous device-status API with roaming and connectivity endpoints has been split into 4 specific APIs
```

**Comment by eric-murray** [comment]
Github is not letting me make suggestions for the "Scope" section in the README, but I'd suggest putting the different functions in different bullet points. e.g.:

* It provides the customer with the ability to:  
  * check if a device is reachable or has lost connection to the network
  * check the roaming status of a device, and which network it is roaming on
  * receive notifications if the status of the device changes

**Comment by hdamker** [comment]
> Github is not letting me make suggestions for the "Scope" section in the README, but I'd suggest putting the different functions in different bullet points

The reason is that these lines are not changed by the PR. As this is also not within the scope of this PR, how about propose it in an issue and address, when some more Codeowners are again available? (it can perfectly done between RC and public release)

**Review by eric-murray — APPROVED** [review]

**Pull request merged**
